### ISSUE TITLE
FPS plumits to zero whilst using a furnace

### ISSUE THREAD

**Issue by wnbrn** [AllTheMods/ATM-6#2078]
I was playing the pack perfectly fine at around 100fps until I started to use a furnace. As soon as I started to smelt some iron, my frames dropped to zero, and I have no clue why this is.  I have and RTX 2060 super, intel I7 9700F, 16GB 2666MHZ RAM so I think I should be able to spelt some iron. i have optifine installed so im unsure if this could effect the particles or something with this modpack. can anyone help me as this is my first time using this mod pack and i really want to play it but i cant due to this fps issue that causes my game to crash.

**Comment by LilHaramBae** [comment]
Having the same issues too!

Doing a 3x ore from mekanism, using **Iron Furnaces' Vibranium Furnace** to smelt the resources and it's causing massive lag on that particular chunk that furnace is in. Thought it was the logistical transporters and the machines, did some testing, nope, then I switched the vibranium furnace with an Ultimate Smelting Factory and everything went back to normal.

In my case, I assume it's because the furnace keeps turning off and on every tick, which then causes the lag (I got no clue, just the assumption).  

My specs are 5950x, 3090, 64GB ram. So that's definitely not the issue lol.
